packages/components/src/alignment-matrix-control/types.ts

```diff
@@ -0,0 +1,50 @@
+export type AlignmentMatrixControlValue =
+	| 'top left'
+	| 'top center'
+	| 'top right'
+	| 'center left'
+	| 'center'
+	| 'center center'
+	| 'center right'
+	| 'bottom left'
+	| 'bottom center'
+	| 'bottom right';
+
+export type AlignmentMatrixControlProps = {
+	/**
+	 * Accessible label. If provided, sets the `aria-label` attribute of the
+	 * underlying <Composite/> component.
+	 *
+	 * @default 'Alignment Matrix Control'
+	 */
+	label?: string;
+	/**
+	 * If provided, sets the default alignment value.
+	 */
+	defaultValue?: AlignmentMatrixControlValue;
+	/**
+	 * The current alignment value.
+	 */
+	value?: AlignmentMatrixControlValue;
+	/**
+	 * A function that receives the updated alignment value.
+	 */
+	onChange?: ( newValue: AlignmentMatrixControlValue ) => void;
+	/**
+	 * If provided, sets the width of the wrapper <Composite/> component.
```

Comment:
Same here. Something like this feels sufficient and more understandable to me:

```suggestion
	 * If provided, sets the width of the control.
```

Reply:
👍